admission-controller/values.yaml

```diff
@@ -70,9 +71,9 @@ deployment:
 
 proxy-scanner:
   fullnameOverride: lacework-proxy-scanner
-  default_registry: index.docker.io
   enabled: true
   config:
+    default_registry: index.docker.io
```

Comment:
Has the corresponding parsing changed in AC code?

Reply:
No, realized that this parameter was in the wrong place before, proxy-scanner wasn't able to read it when it was outside `config`.